src/main/java/appeng/core/definitions/AEItems.java

```diff
@@ -253,6 +249,7 @@
     public static final ItemDefinition<ReplicatorCardItem> DEBUG_REPLICATOR_CARD = item(AppEng.makeId("debug_replicator_card"), ReplicatorCardItem::new);
     public static final ItemDefinition<DebugPartPlacerItem> DEBUG_PART_PLACER = item(AppEng.makeId("debug_part_placer"), DebugPartPlacerItem::new);
     public static final ItemDefinition<FluidDummyItem> DUMMY_FLUID_ITEM = item(AEItemIds.DUMMY_FLUID_ITEM, FluidDummyItem::new);
+    public static final ItemDefinition<TestPatternItem> TEST_PATTERN_ITEM = item(AppEng.makeId("test_pattern"), TestPatternItem::new);
```

Comment:
Can probably get rid of this now that we have processing pattern encoding with fluids,